src/liquid/components/ld-card/readme.md

```diff
@@ -61,6 +69,16 @@ The `size` prop effects the card padding only.
   <LdTypo>Lorem ipsum dolor sit amet.</LdTypo>
 </LdCard>
 
+<LdCard size="zero-padding">
+  <img
+    style={ {
+      borderRadius: 'var(--ld-card-border-radius)'
+    } }
```

Comment:
I'd rather prefer if this was not necessary. Can't we add overflow hidden to the card?

Reply:
I would like to do without `overflow: hidden` where possible. It tends to be quite annoying when you need to have something, that reaches outside of the container. But I'll apply a change to the code which will allow setting border-radius to inherit. This will facilitate the usage of utility classes.